pilot/helpers/agents/Developer.py

```diff
@@ -137,39 +167,45 @@ def step_human_intervention(self, convo, step: dict):
                     'r': lambda conv: run_command_until_success(conv,
                                                                 self.run_command,
                                                                 command_id='app',
-                                                                timeout=None,
+                                                                timeout=1000,
```

Comment:
how come you put timeout when human asks to run command?

Reply:
We want to wait a moment to see if it crashes or not, but if we don't specify a timeout, you have to press CTRL+C to move forward. This allows the user to move forward with the conversation while the app runs in the background.

There's still an issue with the LLM occasionally saying "check the output on the terminal when you send a request from the browser/Postman" - #236